renovate-presets/default.json

```diff
@@ -3,22 +3,21 @@
   "description": "Default preset for use with Jellyfin's repos",
   "extends": [
     "config:base",
-    "helpers:pinGitHubActionDigests",
-    ":rebaseStalePrs",
+    ":disableDependencyDashboard",
+    ":enableVulnerabilityAlertsWithLabel(security)",
+    ":prHourlyLimit2",
+    ":prConcurrentLimit20",
     ":semanticCommits",
     ":semanticCommitScope(deps)",
+    ":timezone(Etc/UTC)",
     "docker:enableMajor",
     "group:testNonMajor",
-    "group:monorepos"
+    "group:monorepos",
+    "helpers:pinGitHubActionDigests"
   ],
   "addLabels": ["dependencies"],
-  "dependencyDashboard": false,
   "internalChecksFilter": "strict",
-  "prHourlyLimit": 2,
   "rebaseWhen": "conflicted",
```

Comment:
Isn't a good idea to do this whenever the PR is behind the base branch? So all CI results are always up to date with the dependency updates.

Reply:
if we where to default too `behind-base-branch` then that would be very noisy no? as not every preset groups every dependency and every rebase/force push to a update PR triggers a notification, no?
You can always overwrite it in the used config in the end, this really is just the general global preset.